FhirToDataLake/src/Microsoft.Health.Fhir.Synapse.Core/Jobs/AzureBlobJobStore.cs

```diff
@@ -260,5 +267,58 @@ private async Task<T> FromBlob<T>(string blobName, CancellationToken cancellatio
                 throw new StartJobFailedException($"Parse blob file '{blobName}' failed.", ex);
             }
         }
+
+        public async Task CommitJobDataAsync(Job job, CancellationToken cancellationToken = default)
+        {
+            var stagingFolder = $"{AzureStorageConstants.StagingFolderName}/{job.Id}";
+            var directoryPairs = new List<Tuple<string, string>>();
+
+            var pathItems = await _blobContainerClient.ListPathsAsync(stagingFolder, cancellationToken);
```

Comment:
If the staging folder contains many results, do we need to return results in batch?

Reply:
Thanks, fixed